lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -150,12 +150,19 @@ def primNameTextChanged(primName):
         if validatedName != primName:
             cmds.textFieldGrp('primNameText', edit=True, text=validatedName)
 
-def variantOrNewPrim(buttonChecked):
-    # The variant and new child prim radio buttons are part of a collection.
+def variantOrNewPrim(variantSelected):
+    # The variant and new child prim radio buttons are mutually exclusive.
     # So only one can be checked.
-    variantSelected = cmds.radioButtonGrp('variantRadioBtn', query=True, select=1)
+    if variantSelected:
+        cmds.radioButtonGrp('variantRadioButton', edit=True, select=True)
+        cmds.radioButtonGrp('newChildPrimInvisibleRadioButton', edit=True, select=True)
+    else:
+        cmds.radioButtonGrp('variantInvisibleRadioButton', edit=True, select=True)
+        cmds.radioButtonGrp('newChildPrimRadioButton', edit=True, select=True)
```

Comment:
The **select** flag is an integer (not a boolean) for which button to select.

Reply:
The python API accept either boolean or int as the value. I can pass 1 instead.